db/migrate/20160916063059_parse_double_encoded_json.rb

```diff
@@ -0,0 +1,18 @@
+class ParseDoubleEncodedJson < ActiveRecord::Migration
+  def change
+    ids = CustomField.where(data_type: 'breakeven_data').ids.map(&:to_s)
+    puts "CustomField ids: #{ids.inspect}"
+    ids.each do |id|
+      lrss = LoanResponseSet.select { |i| i.custom_data.keys.include? id }
+      puts "LoanResponseSet ids: #{lrss.map(&:id).inspect}"
+      lrss.each do |lrs|
+        value = lrs.custom_data[id]['breakeven_data']
+        if value.is_a? String
+          puts "Parsing LRS ##{lrs.id.inspect}"
+          lrs.custom_data[id]['breakeven_data'] = JSON.parse value
+          lrs.save!
+        end
+      end
+    end
+  end
```

Comment:
Nice job on this. I presume you are confident that it works as intended. I'm not going to review the logic in detail. Main thing is you've run it and inspected the results (via the DB console so you can be sure Rails isn't interfereing) and they're correct.